growsocial/common/methods/errorLogsMethods.js

```diff
@@ -1,10 +1,77 @@
 Meteor.methods({
-  addErrorLog: function(error) {
-    ErrorLogs.insert({
-      userId: Meteor.userId(), // null if not logged in
-      dateTime: new Date(),
-      tag: error.tag,
-      message: error.message
-    });
-  },
+	// check: function(object, type){
+	// 	switch(type){
+	// 		case "number":
+	// 		break;
+	// 		case "string":
+	// 		break;
+
+	// 	}
+	// },
+
+	addErrorLog: function(error) {
+
+	  	var err = {
+	  		userId: Meteor.userId(), // null if not logged in
+	  		dateTime: new Date()
+	  	};
+
+	  	// Tag (Type of error) (Max 30 characters)
+	  	if (typeof error.tag === 'string') {
+	  		if (error.tag.length > 30) {
+	  			err.tag = error.tag.substring(0, 31);
+	  		}
+	  		else {
+	  			err.tag = error.tag;
+	  		}
+	  	}
+
+	  	// Message (Max 40 characters)
+	  	if (typeof error.message === 'string') {
+	  		if (error.message.length > 40) {
```

Comment:
Considering we will log errors from other parts of the app, we will need to be more generous here, say 200 characters.